packages/node_modules/@ciscospark/react-component-activity-list/src/index.js

```diff
@@ -29,7 +29,7 @@ export default function ActivityList(props) {
       );
     }
     case ITEM_TYPE_ACTIVITY: {
-      const {activity, avatarUrl, isPending, isAdditional, isFlagged, isSelf} = visibleActivity;
+      const {activity, avatarUrl, isError, isPending, isAdditional, isFlagged, isSelf} = visibleActivity;
```

Comment:
What do you think about using destructuring on line 17 to set the values there instead of doing it for every case?